crates/core/agent_interface/Cargo.toml

```diff
@@ -1,15 +1,17 @@
 [package]
-name = "json_sm"
-version = "0.5.1"
+name = "agent_interface"
+version = "0.5.0"
 authors = ["thin-edge.io team <info@thin-edge.io>"]
 edition = "2018"
 license = "Apache-2.0"
-description = "json_sm defines the thin-edge json extension for software management"
+description = "agent_interface defines all the mqtt topics and messages to be used to invoke the agent."
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
 csv = "1.1"
+download = { path = "../../common/download" }
+mqtt_client = { path = "../../common/mqtt_client" }
```

Comment:
Not a request for change, just a question. I wonder if having a dependency to `mqtt_client` is a good idea, knowing that this is just to use `Topic` vs `String` for the topics.

* Would the change to unchecked `String` topics implies a lot of refactoring ?

Reply:
To create `mqtt topics`  from the `Strings topics` the mqtt_client crate is used here and used in mapper and some tests in agent-interface crate. To remove this the mqtt topic creation can be moved into the mapper directly.